src/main/java/gregtech/common/metatileentities/multi/electric/generator/MetaTileEntityLargeCombustionEngine.java

```diff
@@ -81,34 +86,36 @@ protected void addDisplayText(List<ITextComponent> textList) {
         }
 
         builder.addFuelNeededLine(recipeLogic.getRecipeFluidInputInfo(), recipeLogic.getPreviousRecipeDuration())
```

Comment:
This line does not disappear when the Combustion Engine is no longer running due to filling its output dynamo.

More specifically, the `workingStatus` line is incorrect, as it displays "Running Perfectly", but the Combustion Engine is not running.